env.yml

```diff
@@ -6,6 +6,11 @@ development:
   USERPOOL_APP_CLIENT_NAME: "texvet_development_api"
   USERPOOL_ARN: "arn:aws:cognito-idp:us-east-1:156490493568:userpool/us-east-1_rxLVspG6X"
   SENDGRID_API_KEY: ""
+  EMAIL_SMTP_HOST: "smtp.gmail.com"
```

Comment:
We're using SendGeid to send email but that is not important for this task. This code, should not attempt to send an email it should just set the functions (controller and service) that accept a request to send an email and return a status code and message to the calling service. There should be no reason to configure email at this point.